### PR TITLE
Fix some minor grammatical errors

### PR DESCRIPTION
**Pull request by imReset**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

**2. This PR:**

**3. Additional Information:**

### CODE REVIEW

File: html_css/intermediate_html/Introduction.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ### Introduction
 
-This is the first of the two part course where we dig deeper into HTML and CSS.
+This is the first of the two-part course where we dig deeper into HTML and CSS.
 
-Our foundations lessons, by design, only scratched the surface of what's possible. We wanted to give you all of the bare necessities for getting started so you could move on and be somewhat productive as quickly as possible, but now it's time to slow down and dig in!
+Our foundations lessons, by design, only scratched the surface of what's possible. We wanted to give you all of the bare necessities for getting started, so you could move on and be somewhat productive as quickly as possible, but now it's time to slow down and dig in!
```
Comment (fortypercenttitanium) [inline html_css/intermediate_html/Introduction.md:5]: This comma is unnecessary

File: html_css/intermediate_html/Introduction.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
-You have probably realized by now that there are many more HTML elements than we mentioned in our foundations content. Throughout this course we'll be hitting the rest of the important elements such as forms and tables.
+You have probably realized by now that we mentioned many more HTML elements in our foundation's content. Throughout this course, we’ll hit the rest of the important elements, such as forms and tables.
```
Comment (fortypercenttitanium) [inline html_css/intermediate_html/Introduction.md:7]: This changes the meaning of the sentence. We're saying there are more elements that we *haven't* covered yet. Your sentence implies that we covered more elements in the foundations lesson, which is not the case.

**Review by fortypercenttitanium — CHANGES_REQUESTED** [review]
Just a few changes and we'll get it merged! Thanks!

File: html_css/intermediate_html/Introduction.md (hunk 2 above)
Comment (chayboats) [inline html_css/intermediate_html/Introduction.md:7]:
```suggestion
You have probably realized by now that we mentioned many more HTML elements in our foundation's content. Throughout this course, we’ll cover the rest of the important elements, such as forms and tables.
```

**Pull request closed without merging**